atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/MultiCassandraStartupOrderingEteTest.java

```diff
@@ -68,56 +69,45 @@ public void randomizeKeyspace() throws IOException, InterruptedException {
                 + "/' var/conf/atlasdb-ete.yml");
     }
 
-    @AfterClass
-    public static void resetKeyspace() throws IOException, InterruptedException {
-        EteSetup.execCliCommand("sed -i 's/keyspace: .*/keyspace: atlasete/' var/conf/atlasdb-ete.yml");
-        stopTheAtlasServer();
-        startTheAtlasServer();
+    @After
+    public void restoreCassandra() throws InterruptedException {
+        startCassandraNodes();
     }
 
     @Test
     public void shouldBeAbleToStartUpIfCassandraIsDownAndRunTransactionsWhenCassandraIsUp()
             throws IOException, InterruptedException {
-        stopTheAtlasServer();
-
-        stopCassandraNodes();
-
-        startTheAtlasServer();
-
-        assertNotSatisfiedWithin(40, MultiCassandraStartupOrderingEteTest::canAddTodo);
+        restartAtlasWithDegradedCassandra();
+        assertNotSatisfiedWithin(40, MultiCassandraStartupOrderingEteTest::canPerformTransaction);
 
         startCassandraNodes();
+        assertSatisfiedWithin(60, MultiCassandraStartupOrderingEteTest::canPerformTransaction);
 
-        assertSatisfiedWithin(150, MultiCassandraStartupOrderingEteTest::canAddTodo);
-
-        stopTheAtlasServer();
+        restartAtlasWithDegradedCassandra();
+        Assert.assertEquals(hasQuorum(), canPerformTransaction());
+    }
 
+    private void restartAtlasWithDegradedCassandra() throws IOException, InterruptedException {
+        stopAtlasServer();
         stopCassandraNodes();
-
         startTheAtlasServer();
-
-        if (hasQuorum()) {
-            assertSatisfiedWithin(120, MultiCassandraStartupOrderingEteTest::canAddTodo);
-        } else {
-            assertNotSatisfiedWithin(40, MultiCassandraStartupOrderingEteTest::canAddTodo);
-        }
     }
 
-    private static void stopTheAtlasServer() throws IOException, InterruptedException {
+    private static void stopAtlasServer() throws IOException, InterruptedException {
         EteSetup.execCliCommand("service/bin/init.sh stop");
-        assertSatisfiedWithin(10, () -> !serverStarted());
+        assertSatisfiedWithin(10, () -> !serverRunning());
     }
 
     private static void startTheAtlasServer() throws IOException, InterruptedException {
```

Comment:
nit: For consistency with the stop-method I think we should name this `startAtlasServer()`.